samtools/samtools_stats.cwl

```diff
@@ -0,0 +1,149 @@
+class: CommandLineTool
+cwlVersion: v1.0
+
+baseCommand: [samtools, stats]
+requirements:
+- class: DockerRequirement
+  dockerPull: biocontainers/samtools:v1.7.0_cv3
+
+inputs:
+  input_file:
+    type: File
+     format: 
+      - edam:format_2572  # BAM
+      - edam:format_2573  # SAM
+      - edam:format_3462  # CRAM
```

Comment:
Have you tested this? `samtools stats --help` says "The program collects statistics from BAM files."

Reply:
I tested on SAM and BAM files, not on CRAM file, but in the doc it is said "samtools stats [options] in.sam|in.bam|in.cram [region...] "

Reply:
I think you have a different samtools version than the one in the Docker container:
```
$ docker run -it biocontainers/samtools:v1.7.0_cv3 samtools stats --help
About: The program collects statistics from BAM files. The output can be visualized using plot-bamstats.
Usage: samtools stats [OPTIONS] file.bam
       samtools stats [OPTIONS] file.bam chr:from-to
```

Reply:
Oh, I don't know, I tested with this cwl file and this container on SAM and BAM

Reply:
Lets keep all three for now. It seems likely that CRAM works if SAM & BAM do as well.